### ISSUE TITLE
Add CI for multi platform docker builds

### ISSUE THREAD

**Issue by joepio** [atomicdata-dev/atomic-data-rust#198]
Would be awesome if 32 / 64 / armv7 docker image builds are made in the CI

**Comment by joepio** [comment]
Seems to fail: `buildx call failed with: error: invalid tag "joepmeneer/atomic-server:-195-parse-nested-resources": invalid reference format`

Not sure why this action https://github.com/docker/build-push-action fails.

I'll try a different version

**Comment by joepio** [comment]
Maybe the name of the tag is wrong, because it starts with a dash. But it's weird that it's using that tag at all - I never set it. Maybe I should re-push some other tag, and prevent manually running this task.

**Comment by joepio** [comment]
Cargo-chef not working because... It can't reach crates.io?

```
> [linux/arm/v7 planner 3/5] RUN cargo install cargo-chef:
#51 2.251     Updating crates.io index
#51 2.568 warning: spurious network error (2 tries remaining): could not read directory '/usr/local/cargo/registry/index/github.com-1285ae84e5963aae/.git//refs': Value too large for defined data type; class=Os (2)
#51 2.785 warning: spurious network error (1 tries remaining): could not read directory '/usr/local/cargo/registry/index/github.com-1285ae84e5963aae/.git//refs': Value too large for defined data type; class=Os (2)
#51 2.997 error: failed to fetch `https://github.com/rust-lang/crates.io-index`
#51 2.997 
#51 2.997 Caused by:
#51 2.998   could not read directory '/usr/local/cargo/registry/index/github.com-1285ae84e5963aae/.git//refs': Value too large for defined data type; class=Os (2)
------
dockerfile:10
--------------------
   8 |     FROM rust  as cacher
   9 |     WORKDIR /app
  10 | >>> RUN cargo install cargo-chef
  11 |     COPY --from=planner /app/recipe.json recipe.json
  12 |     RUN cargo chef cook --release --recipe-path recipe.json
--------------------
error: failed to solve: process "/bin/sh -c cargo install cargo-chef" did not complete successfully: exit code: 101
Error: buildx failed with: error: failed to solve: process "/bin/sh -c cargo install cargo-chef" did not complete successfully: exit code: 101
```

**Comment by joepio** [comment]
It's probably this issue: https://github.com/rust-lang/cargo/issues/8719

**Comment by joepio** [comment]
The new action works! Just built the first armv7 image on github CI.
